src/centos/taskset_boot.sh.j2

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/usr/bin/taskset -acp {{ cpusystem }} 2
+/usr/bin/taskset -acp {{ cpusystem }} $(pgrep rcu_preempt)
```

Comment:
I think it is already done by tunectl, isn't?

Reply:
removed